terraform/deployments/cluster-services/kube_prometheus_stack.tf

```diff
@@ -96,6 +97,45 @@ resource "helm_release" "kube_prometheus_stack" {
           }
         }
       }
+      serviceAccount = {
+        annotations = {
+          "eks.amazonaws.com/role-arn" = local.grafana_iam_role
+        }
+      }
+      env = {
+        "AWS_ROLE_ARN"                = local.grafana_iam_role
+        "AWS_WEB_IDENTITY_TOKEN_FILE" = "/var/run/secrets/eks.amazonaws.com/serviceaccount/token"
+        "AWS_REGION"                  = data.aws_region.current.name
+      }
+      extraSecretMounts = [
+        { name      = "aws-iam-token"
```

Comment:
nit: what's up with the indentation here? is this what `terraform fmt` produced?! 😅

Reply:
I can try to adjust it, but I did run fmt on it.

Reply:
Weird. If it's a pain then don't sweat it 😅